src/compute/flexbox.rs

```diff
@@ -628,6 +628,7 @@ fn determine_flex_base_size(
 
         // Parent size for child sizing
         let cross_axis_parent_size = constants.node_inner_size.cross(dir);
+        let child_parent_size = Size::NONE.with_cross(dir, cross_axis_parent_size);
```

Comment:
Not really related to this PR, but perhaps we can introduce something like `Size::cross` in a later PR instead of using this pattern :)